src/utils/dispute-utils.js

```diff
@@ -271,23 +284,24 @@ export function getAdjudicationPhase(dispute, round, date, courtConfig) {
     }
   }
 
-  // If given term is before the reveal start term of the last round, then jurors are still allowed to commit votes for the last round
-  const revealTermStartTime = draftTermEndTime + commitTerms * termDuration
-  if (date.isBefore(revealTermStartTime)) {
+  // Jurors can commit their votes between when the commit phase term starts and ends
+  // Note that the commit start term is the same as the draft end term
+  const commitEndTerm = draftEndTerm + commitTerms - 1
```

Comment:
Saw the note above, but wanted to sanity and double check the `- 1`.

If the `draftEndTerm` is `3`, and we allow for `2` `commitTerms`, then we would be allowed to start committing votes at `3` until `4`?

It just seems a bit weird _this_ particular phase is _inclusive_, whereas all the other seem _exclusive_.

Reply:
> If the draftEndTerm is 3, and we allow for 2 commitTerms, then we would be allowed to start committing votes at 3 until 4?

Yes, this is correct. I think that the reason works this way is because the drafting phase can be executed any time as opposed to other phases in which there's a fixed period.

See https://github.com/aragon/aragon-court/blob/492abfd3fb887ab5dd9a71dc480ef7e33c45b7d1/contracts/disputes/DisputeManager.sol#L1054

More interestingly, in your example, if the drafting is executed say at the middle of the term, then the total commit period would last 1 and a half terms.

Reply:
Ah yes, I remember this being discussed and I may have confused the evidence phase with the draft phase from this change: https://github.com/aragon/aragon-court/commit/68587d72337fdd56ce955a65d6231bc17dff5952.

But yes, this makes sense; selected jurors should be able to immediately commit votes!